docs/treasury.md

```diff
@@ -37,10 +39,12 @@ When a proposal is rejected, the returned amount is computed as `slash_scale_val
 
 ### Decision lambda
 
-The decision lambda makes all requested transfers one by one.
-In case of insufficient balance the decision lambda can fail.
-  + For XTZ transfers it sends `amount` to the `recepient` address with `unit` parameter. I. e. it returns a single `TRANSFER_TOKENS` operation.
-  + For FA2 transfers it calls `transfer` entrypoint of the `fa2` contract with given argument.
+For the Transfer proposal, it makes all requested transfers one by one.
+ - In case of insufficient balance the decision lambda can fail.
+    + For XTZ transfers it sends `amount` to the `recepient` address with `unit` parameter. I. e. it returns a single `TRANSFER_TOKENS` operation.
+    + For FA2 transfers it calls `transfer` entrypoint of the `fa2` contract with given argument.
+
+For the Update guardian proposal, it updates the guardian address in the storage.
```

Comment:
I guess this same change should be done in the `registry` document too?

Reply:
Ah, yes. Added now.